engine/src/test/java/org/entando/entando/web/digitalexchange/DigitalExchangesControllerIntegrationTest.java

```diff
@@ -148,9 +171,21 @@ public void shouldFailDeletingDigitalExchangeBecauseNotFound() throws Exception
         result.andExpect(jsonPath("$.payload").isEmpty());
     }
 
+    @Test
+    public void shouldTestInstance() throws Exception {
+
+        ResultActions result = createAuthRequest(get(BASE_URL + "/test/{name}", "DE 1")).execute();
+
+        result.andExpect(status().isOk());
+        result.andExpect(jsonPath("$.metaData").isEmpty());
+        result.andExpect(jsonPath("$.errors").isEmpty());
+        result.andExpect(jsonPath("$.payload", is("OK")));
```

Comment:
Nothing wrong here, but if you're feeling lazy next time:

```
        result.andExpect(status().isOk())
                .andExpect(jsonPath("$.metaData").isEmpty())
                .andExpect(jsonPath("$.errors").isEmpty())
                .andExpect(jsonPath("$.payload", is("OK")));

```

Reply:
thanks for the suggestion